pkg/plugin/plugin.go

```diff
@@ -404,7 +403,8 @@ func CmdAdd(args *skel.CmdArgs) error {
 				return fmt.Errorf("failed to look up %q: %v", args.IfName, err)
 			}
 			for _, ipc := range newResult.IPs {
-				if ipc.Version == "4" {
+				// if ip address version is 4
+				if ipc.Address.IP.To4() != nil {
```

Comment:
Please comment in the commit message why this had to change with bump of CNI.

Reply:
Ok, fixed.